content/docs/aro/cert-manager/_index.md

```diff
@@ -80,8 +43,15 @@ I'm going to be running this setup through Fedora in WSL2. Be sure to always use
 
 1. Browse to https://cloud.redhat.com/openshift/install/azure/aro-provisioned
 
-1. click the **Download pull secret** button and remember where you saved it, you'll reference it later.
+1. click **Copy pull secret** 
+
+1. Back in the Azure Cloud Shell, enter the following. Insetad of `paste_your_secret`, paste your actual secret. To do so, right-click and choose paste.
 
+    ```bash
+    cat >> ./pull-secret.txt << EOF
+    paste_your_secret
```

Comment:
I would change this to `<paste_your_secret>`  as its more obvious they need to change things when you have it surrounded.